lib/elixir/lib/io.ex

```diff
@@ -250,7 +250,7 @@ defmodule IO do
   Reads a line from the IO device. It returns:
 
     * `data` - the characters in the line terminated
-      by a LF (or end of file)
+      by an LF (or end of file)
```

Comment:
I changed it a bit, to be more consistent.

    * `data` - the characters in the line terminated
      by a line-feed (LF) or end of file (EOF)